### PR TITLE
Student Commit

### PR DESCRIPTION
**Pull request by Sagar-Fexle-May2022**

### CODE REVIEW

File: SRC/classes/Register.cls [hunk 1]
```diff
@@ -0,0 +1,52 @@
+public class Register {
```
Comment (sushilchoudhary7) [inline SRC/classes/Register.cls:1]: Fix indentation of this class.

File: SRC/classes/RegisterDebug.cls [hunk 2]
```diff
@@ -0,0 +1,13 @@
+public class RegisterDebug {
```
Comment (sushilchoudhary7) [inline SRC/classes/RegisterDebug.cls:1]: Fix indentation of this class.

File: SRC/classes/Teacher.cls [hunk 3]
```diff
@@ -0,0 +1,49 @@
+public class Teacher {
```
Comment (sushilchoudhary7) [inline SRC/classes/Teacher.cls:1]: Fix indentation of this class.

File: SRC/classes/TeacherDebug.cls [hunk 4]
```diff
@@ -0,0 +1,15 @@
+public class TeacherDebug {
```
Comment (sushilchoudhary7) [inline SRC/classes/TeacherDebug.cls:1]: Fix indentation of this class.

**Review by Sagar-Fexle-May2022 — COMMENTED** [review]
All Indention are fixed

**Review by sushilchoudhary7 — APPROVED** [review]

**Pull request merged**